extensions/v1/smi/spec.yaml

```diff
@@ -7,7 +7,7 @@ shortDescription: |
   The Service Mesh Interface (SMI) is a specification for service meshes that run on Kubernetes. It defines a common standard that can be implemented by a variety of providers.
 versions:
 - manifestsArchive:
-    uri: https://storage.googleapis.com/sm-marketplace-registry/smi.tar.gz
+    uri: https://storage.googleapis.com/sm-marketplace-registry/smi-0.1.0.tar.gz
```

Comment:
while we're here could you update the pod spec in the manifest archive to point to image tag `0.1.0` rather than `latest`? https://hub.docker.com/r/deislabs/smi-adapter-istio/tags

Reply:
Done. Also removed the istio-system namespace requirement. It still needs to be installed to the same namespace as the istio mesh, though.